File: src/csvParse.nim
import parsecsv, strutils

const
  changeRow = 2
  changeColumn = 4
  changeValue = "\"Spam\""
var
  reader: CsvParser
  lineNum = 1

reader.open("input.csv")
while reader.readRow():
  if lineNum == changeRow:
    reader.row[changeColumn - 1] = changeValue
  let line = reader.row.join(",") & "\n"
  echo line
  lineNum += 1

File: src/csv_parse.py
import csv

change_row, change_column, change_value = 2, 4, "\"Spam\""
line_num = 1

reader = csv.reader(open('input.csv'), delimiter=',')
for fields in reader:
    if line_num == change_row:
        fields[change_column - 1] = change_value
    line = ','.join(fields) + '\n'
    print(line)
    line_num += 1
